docs/nft-support/royalties-and-fees.rst

```diff
@@ -0,0 +1,48 @@
+Generating 0x Order Hashes
```

Comment:
this doesn't appear to concern NFT royalties, but instead how to hash the limit order struct

Reply:
Good catch, updated this page